### PR TITLE
Configure linters and Add HTML and CSS files

### PR DESCRIPTION
**Pull request by Lasten-Ernest**
This pull request is for adding HTML and CSS files and configuring the linters which are used for conducting automated tests on the html syntax and CSS among others.

### CODE REVIEW

File: FirstProject.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (rustam-tolipov) [inline FirstProject.html]: - [x] Good job so far 🙌🏼. Please always use lower case in your file names. I would suggest changing HTML to `index.html` and CSS to `style.css`.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,97 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (rustam-tolipov) [inline README.md:1]: - [ ] In your reamde, please use the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and change it based on your project 👍🏼

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Lasten-Ernest , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Lasten-Ernest) [reply README.md:1]: Thanks for the comments.

I have fixed the issues now.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Lasten-Ernest  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Fixed changes requested by previous reviewer 🏄 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**